### ISSUE TITLE
Keep a dinosaur alive

### ISSUE THREAD

**Issue by ESloman** [ESloman/bsebot#267]
**Is your feature request related to a problem? Please describe.**
Pook thinks we should have a tamagotchi type dino we keep alive via eddies.
The person who spends the most eddies on keeping it alive is it's favourite.

**Comment by brods** [comment]
I would quite like to write some stuff for this maybe. Could make it a bit like a game (BSE Games strikes again). I'll try and spend some time fleshing it out soon so you get an idea of what might be required code-wise or (maybe!) I can try and learn some like simple logic code if you show me some bits about how the bot works haha

**Comment by ESloman** [comment]
Yeah sounds good :) I wasn't entirely sure what _would_ be needed or how it would work so happy to hear your idea fleshed out!
Also happy to show you the ropes a bit so you can learn so we'll set that up at some point!
